src/CoreWCF.Primitives/src/CoreWCF/Security/SecuritySessionServerSettings.cs

```diff
@@ -24,7 +25,7 @@ internal sealed class SecuritySessionServerSettings : IServiceDispatcherSecureCo
         internal static readonly TimeSpan defaultKeyRolloverInterval = TimeSpan.FromSeconds(5);
         internal const bool defaultTolerateTransportFailures = true;
         internal const int defaultMaximumPendingSessions = 128;
-        internal static readonly TimeSpan defaultInactivityTimeout = TimeSpan.FromSeconds(2);
+        internal static readonly TimeSpan defaultInactivityTimeout = TimeSpan.FromSeconds(10);
```

Comment:
Both the original and this are wrong. Also defaultKeyRolloverInterval and defaultKeyRenewalInterval are also wrong. See here where this should be 2 **minutes** and the key rollover should be 5 **minutes**. Also the KeyRenewalInterval should be 15 **hours**.

Reply:
forgot to update.. all values you see were relevant to my local testing... thanks for pointing that..